internal/learn/future/assembly.md

```diff
@@ -0,0 +1,27 @@
+---
+id: assembly
+title: Assembly
+description: Introduction to the emerging permissionless Smart Contracts network
+keywords:
+  - smart contracts
+  - permissionless
+  - ASMB
+---
+
+![Assembly](/img/learn/future/assembly.png 'Click to see the full-size image.')
+
+In December 2021, IOTA Foundation [introduced](https://blog.assembly.sc/announcing-assembly-and-the-asmb-token/) Assembly, a permissionless Smart Contracts network. Assembly will function as an intermediate layer for permissionless and interoperable Smart Contracts, and will have its own token called [ASMB](https://assembly.sc/token).
+For more information, check the official [Website](https://assembly.sc) or the [Assembly Wiki](https://wiki.assembly.com).
+
+# The role of the IOTA Token
+
+So you maybe asked yourself what the role of IOTA is in this Assembly network. Assembly is build on top of the Tangle, it will need the Tangle for anchored state updates, interoperability between chains, for Native Assets and general access to a congested network. So there are basically two main usages of the IOTA Token in Assembly.
+
+## The new dust protection
+
+With the new [dust protection](https://github.com/muXxer/protocol-rfcs/blob/master/text/0032-dust-protection/0032-dust-protection.md), every data stored on the Tangle needs to be backed by IOTA Tokens. So every NFT, SC state update, Native Asset requires IOTA Tokens so that it can be stored on L1. You can also read more about the new dust protection in this [community blog post](https://medium.com/@wernerderchamp/dust-protection-on-the-iota-network-an-eli12-d8ca567a2d36).
+
+## Mana
```

Comment:
What if we rearrange this paragraph?

1. Assembly nodes want to anchor their state even if Tangle is congested
2. Right now Coordinator manages traffic, but after coordicide you will need Mana (link to Mana)
3. To generate mana, you need to own IOTA tokens
4. It means that ASMB does not compete with MIOTA, but creates demand for it.

Reply:
You mean rearrange only this Mana paragraph?
I just ask because point 4 is the general conclusion of Mana and the new dust protection, right? So should that be it‘s own „paragraph“?

Reply:
Good point. You could finish paragraph with something like ", creating even more demand for the IOTA token". So it would include the dust protection and wrap up the whole article.

Reply:
Another question for point 2. The coordinator manages traffic doesn‘t sound right to me. I mean yeah indirectly if the coo crashes he controls troughput :trollface: , but mainly mainnet manages traffic by crashing weak nodes right now 😅 
So maybe it‘s more like: `Currently there is no congestion mechanism in place but after coordicide you will need Mana to get your fair share in a congested network`?

Reply:
Ye. Maybe  don't even mention the current state and just go with "after coordicide/in IOTA 2.0 network nodes will need Mana..."

Reply:
What about now? 😅